addon/ng2/commands/set.ts

```diff
@@ -0,0 +1,25 @@
+import * as Command from 'ember-cli/lib/models/command';
+import {CliConfig} from '../models/config';
+
+
+const SetCommand = Command.extend({
+  name: 'set',
+  description: 'Set a value in the configuration.',
+  works: 'insideProject',
```

Comment:
If it is possible to have a global config, this should command should work even outside of projects.